client/package.json

```
@@ -12,12 +12,14 @@
     "test": "jest --maxWorkers=1"
   },
   "dependencies": {
+    "@types/styled-components": "^5.1.7",
     "babel-jest": "^26.6.3",
     "fuse.js": "^6.4.6",
     "next": "10.0.3",
     "react": "17.0.1",
     "react-dom": "17.0.1",
-    "vercel-is-pull-request": "^0.0.8"
+    "vercel-is-pull-request": "^0.0.8",
+    "styled-components": "^5.2.1"
```

Let's keep alphabetical order

```suggestion
    "styled-components": "^5.2.1",
    "vercel-is-pull-request": "^0.0.8"
```